diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/WarningsTest.kt

```diff
@@ -0,0 +1,13 @@
+package org.cqfn.diktat.ruleset.utils
+
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.junit.jupiter.api.Test
+
+class WarningTest {
+    @Test
+    fun `checking that warnings has all proper fields filled`() {
+        Warnings.values().forEach { warn ->
+            assert(warn.ruleId.split(".").size == 3)
```

Comment:
Maybe, this check can be moved into an `init` block of `Warnings`? This way an incorrect id can be spotted in all other tests too.

Reply:
I actually think that it is fine to have such tests here, not in main logic.
It is like an encapsulation